gpii/node_modules/flowManager/test/PSPChannelTests.js

```diff
@@ -323,57 +319,75 @@ gpii.tests.pspChannel.testDef = {
             }
         }
     },
-    sequence: [{
-        func: "{clientOne}.connect"
-    }, {
-        event: "{clientOne}.events.onConnect",
-        listener: "gpii.tests.pspChannel.connectionSucceeded"
-    }, {
-        event: "{clientOne}.events.onReceiveMessage",
-        listener: "gpii.tests.pspChannel.checkPayload",
-        args: ["{arguments}.0", "connectWithNoUsers"]
-    }, {
-        func: "{loginUser1a}.send"
-    }, {
-        event: "{clientOne}.events.onReceiveMessage",
-        listener: "gpii.tests.pspChannel.checkPayload",
-        args: ["{arguments}.0", "logoutUser"]
-    }, {
-        event: "{clientOne}.events.onReceiveMessage",
-        listener: "gpii.tests.pspChannel.checkPayload",
-        args: ["{arguments}.0", "snapset_1a_loggedIn"]
-    }, {
-        func: "{clientOne}.disconnect"
-    }, {
-        func: "{clientTwo}.connect"
-    }, {
-        event: "{clientTwo}.events.onConnect",
-        listener: "gpii.tests.pspChannel.connectionSucceeded"
-    }, {
-        event: "{clientTwo}.events.onReceiveMessage",
-        listener: "gpii.tests.pspChannel.checkPayload",
-        args: ["{arguments}.0", "snapset_1a_loggedIn"]
-    }, {
-        func: "{logoutUser1a}.send"
-    }, {
-        event: "{clientTwo}.events.onReceiveMessage",
-        listener: "gpii.tests.pspChannel.checkPayload",
-        args: ["{arguments}.0", "logoutUser"]
-    }, {
-        event: "{clientTwo}.events.onReceiveMessage",
-        listener: "gpii.tests.pspChannel.checkPayload",
-        args: ["{arguments}.0", "connectWithNoUsers"]
-    }, {
-        func: "{loginUser2c}.send"
-    }, {
-        event: "{clientTwo}.events.onReceiveMessage",
-        listener: "gpii.tests.pspChannel.checkPayload",
-        args: ["{arguments}.0", "logoutUser"]
-    },  {
-        event: "{clientTwo}.events.onReceiveMessage",
-        listener: "gpii.tests.pspChannel.checkPayload",
-        args: ["{arguments}.0", "snapset_2c_loggedIn"]
-    }]
+    sequence: [
```

Comment:
I imagine it is something in your editor which does this reformatting?

Reply:
I was struggling hard to read all the tests with the indentation, and expressly redid the indentation so that I could reorganise and rewrite the tests more easily.  IMO we should only really collapse an array and an enclosed object into a single level of indentation if the array only has one member.  Anything beyond that and it gets very easy to lose track of where you are, at least for me.